.travis/stop_kuzzle.sh

```diff
@@ -0,0 +1 @@
+docker-compose -f .codepipeline/docker-compose.yml rm -f
```

Comment:
Why not using `docker-compose -f .codepipeline/docker-compose.yml down` ?

Reply:
I think it's better to remove containers than just stop them, to be sure that we have a clean instance of Kuzzle when we launch test

Reply:
`docker-compose down` stop and remove the containers

Reply:
👍 I didn't know it